projects.yaml

```diff
@@ -1928,3 +1935,9 @@ calculated from density functional theory as the objective function."
   license: GPL-3.0
   github_id: jeherr/element-encoder
   # package managers: none
+- name: MODNet
+  category: general-tool
+  description: "MODNet: a framework for machine learning materials properties"
```

Comment:
Will remove `description`. The specified repo `ppdebreuck/modnet` has a description (the same). CI/CD will fetch that. Only provide description if repo does not have (a reasonably informative) one, or if the project is a resource.